canonical/juju.is
# Update navigation

## Done
Updated the navigation

## QA
- Go to https://juju-is-437.demos.haus/
- Check that the navigation looks and works as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-845
